### ISSUE TITLE
Double tap on dialog not starting Text-to-speech feature automatically

### ISSUE THREAD

**Issue by Gomibako7** [BrianJr03/Issa-Chat-App#10]
I have noticed an issue where double tapping on a dialog does not start the Text-to-speech feature automatically. I have tested this on Android 13 (Pixel) with Google Voice Services installed and English US set as the default audio output.

Ty for your awesome app

**Comment by BrianJr03** [comment]
@Gomibako7 This issue has been fixed in the latest version, v1.2
https://github.com/BrianJr03/Issa-AI-App/releases/tag/v1.2

**Comment by BrianJr03** [comment]
@Gomibako7 I just released v1.3 which allows one to save their conversational context. In regards to the TTS issue, is it playing at all or is there a delay before it plays?

**Comment by Gomibako7** [comment]
The conversational context is now working perfectly and being saved, which is great :)

However, I am still experiencing issues with the audio TTS feature. Unfortunately, it is not working at all on my end.

Thanks to you :)

**Comment by BrianJr03** [comment]
@Gomibako7 So I've still been looking into the TTS issue and I'm unsure why it's not working for you. I've installed a Pixel 6 emulator running Android 13 and it worked for me, granted it is an emulator and is not running GrapheneOS. Is it possible for you to run this project in Android Studio and attempt to activate TTS and record and send me the logs from Logcat? I have no way of replicating this issue and this could potentially help. Thanks again for checking out the app, I just released v1.4 btw :)
